seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -205,6 +205,10 @@
       "challengeSeed":[
         "var oldArray = [1,2,3,4,5];",
         "",
+        "var timesFour = oldArray.map(function(val) {",
+        "        return val * 4;",
+        "    });",
+        "",
         "// Only change code below this line.",
```

Comment:
The change in the description should be enough to cover this issue. Please remove this from the seed.